src/highdicom/map/sop.py

```diff
@@ -0,0 +1,789 @@
+        pixel_array: np.ndarray,
+        series_instance_uid: str,
+        series_number: int,
+        sop_instance_uid: str,
+        instance_number: int,
+        manufacturer: str,
+        manufacturer_model_name: str,
+        software_versions: Union[str, Tuple[str]],
+        device_serial_number: str,
+        contains_recognizable_visual_features: bool,
+        real_world_value_mappings: Sequence[RealWorldValueMapping],
+        window_center: Union[int, float],
+        window_width: Union[int, float],
+        transfer_syntax_uid: Union[str, UID] = ImplicitVRLittleEndian,
+        content_description: Optional[str] = None,
+        content_creator_name: Optional[str] = None,
+        pixel_measures: Optional[PixelMeasuresSequence] = None,
+        plane_orientation: Optional[PlaneOrientationSequence] = None,
+        plane_positions: Optional[Sequence[PlanePositionSequence]] = None,
+        **kwargs,
+    ):
+        """
+        Parameters
+        ----------
+        source_images: Sequence[pydicom.Dataset]
+            Sequence of one or more `pydicom.Dataset`s that this parametric map
+            is derived from
+        pixel_array: numpy.ndarray
+            Array of parametric map pixel data of unsigned integer or
+            floating-point data type representing one or more frames of the
+            parametric map pixel data. The values are supposed to represent a
+            single "feature", i.e., be the result of one set of image
+            transformations such that the same `real_world_value_mappings`
+            apply.
+        series_instance_uid: str
+            UID of the series
+        series_number: Union[int, None]
+            Number of the series within the study
+        sop_instance_uid: str
+            UID that should be assigned to the instance
+        instance_number: int
+            Number that should be assigned to the instance
+        manufacturer: str
+            Name of the manufacturer (developer) of the device (software)
+            that creates the instance
+        manufacturer_model_name: str,
+            Name of the model of the device (software)
+            that creates the instance
+        software_versions: Union[str, Tuple[str]]
+            Versions of relevant software used to create the data
+        device_serial_number: str
+            Serial number (or other identifier) of the device (software)
+            that creates the instance
+        contains_recognizable_visual_features: bool
+            Whether the image contains recognizable visible features of the
+            patient
+        real_world_value_mappings: Sequence[highdicom.map.RealWorldValueMapping]
+            Descriptions of how stored values map to real-world values. The
+            concept of real-world values is a bit fuzzy and the mapping may be
+            difficult to describe (e.g., in case of the feature maps of a deep
+            convolutional neural network model).
```

Comment:
Need an explanation about why there may be more than 1, and how this relates to the pixel_array. Also may want to make this a bit more formal in tone?

Reply:
This is now outdated